config/install/field.storage.node.field_cargo_c.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_cargo_c
+field_name: field_cargo_c
```

Comment:
This field machine name doesn't match the content model. Change in Drupal or in the content model so they match.
![starships_field_cargo_capacity_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/d1404353-e324-44e9-b088-68c5d956309f)

Reply:
Fixed